### PR TITLE
Add the possibility of simulating the motion on an inclined surface

### LINKED ISSUE #72: Provide the simulation of the motion on inclined surface
Currently, in the simulator, we assume that the surface is flat. But, it will be useful to extend the simulator for the inclined surfaces.

### PR DESCRIPTION
**Pull request by VenusPasandi**
In this PR, we extend the dynamics core fro the motion on an inclined surface. Note that we just extended the core dynamics not the visualizer. So, we still need to modify the robot visualizer for the inclined surface.

fix #72

### CODE REVIEW

**Comment by traversaro** [comment]
Thanks. Do you think it could make sense to add a line to the https://github.com/ami-iit/matlab-whole-body-simulator/blob/master/CHANGELOG.md ?

**Comment by VenusPasandi** [comment]
> Thanks. Do you think it could make sense to add a line to the https://github.com/ami-iit/matlab-whole-body-simulator/blob/master/CHANGELOG.md ?

Good point. I added a recap of changes to the changeLog.

**Review by traversaro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
 
 ## [Unreleased]
+### Added
+- Added the option to model the motion on an inclined surface (https://github.com/ami-iit/matlab-whole-body-simulator/pull/78).
+
+## [3.0.0] - 2022-04-14
+### Added
+- Added the possibility of simulating robots with cylindrical or spherical feet (https://github.com/ami-iit/matlab-whole-body-simulator/pull/71)
+- Added a new simulator block called `RobotDynWithContacts_closedChain` that is capable of simulating a robot with multiple closed kinematic chains and of defining multiple links of the robot as the links interacting with the ground (https://github.com/ami-iit/matlab-whole-body-simulator/pull/64).
+- Added the possibility of simulating the sliding contact model.
```
Comment (traversaro) [inline CHANGELOG.md:15]: This seems spurious?

Comment (VenusPasandi) [reply CHANGELOG.md]: Sorry it was a typo came form the rebase. I corrected it in c9b383e04eea0e09e3b9b5b069da62bad34599e3

**Review by traversaro — APPROVED** [review]

**Pull request merged**